### PR TITLE
Made a few small improvements to the README

### PR DESCRIPTION
**Pull request by sandlerben**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,26 +71,37 @@ INFO[0000] Profiling ...
...
 
-Expose a pprof endpoint. Official Go docs are [here](https://golang.org/pkg/net/http/pprof/). If your application is already running a server on the DefaultServeMux, just add this import to your application.
+Expose a pprof endpoint. Official Go docs are
+[here](https://golang.org/pkg/net/http/pprof/). If your application is already
+running a server on the DefaultServeMux, just add this import to your
+application.
 
 ```go
-import _ "net/http/pprof"
+import _ "net/http/pprof"
```
Comment (sectioneight) [inline README.md]: it looks like you're adding a bunch of trailing spaces in this diff. is github lying to me?

Comment (sandlerben) [inline README.md]: No, it's correct. I don't know how they got in there. I will clean them up.

**Comment by sandlerben** [comment]
@sectioneight Is it good?

**Comment by sectioneight** [comment]
:shipit:

**Pull request merged**